docs/source/markdown/podman-image-prune.1.md

```diff
@@ -30,7 +30,7 @@ Do not provide an interactive prompt for container removal.
 
 Print usage statement
 
-## Examples ##
+## EXAMPLES ##
```

Comment:
```suggestion
## EXAMPLES
```

Reply:
Maybe GitHub is acting up, but it looks like your latest push didn't remove the trailing ` ##` here?

Reply:
Whoops, I did the old "committed but not pushed" mistake. :sweat_smile: Fixed now.